dashboard/src/router/UseGlobalNavGuards.js

```diff
@@ -92,10 +100,11 @@ export const useGlobalNavGuards = () => {
           p !== '/' ? { name: 'UserAgreement', query: { redirect: p } } : { name: 'UserAgreement' }
         next(ua)
       } else {
-        if (to.path === '/') {
-          const landingPageRoute = { name: getLandingPage() }
-          next(landingPageRoute)
-        }
+       if (to.path === '/' && isLoggedIn()) {
```

Comment:
`isLoggedIn()` was added; suggesting to revert the change

Reply:
Reverted